src/FsAutoComplete.Core/AbstractClassStubGenerator.fs

```diff
@@ -180,67 +179,53 @@ let writeAbstractClassStub
         | _ -> return! None
       }
 
-    let getMemberByLocation (name, range: Range) =
-      asyncOption {
-        let pos = Position.fromZ (range.StartLine - 1) (range.StartColumn + 1)
-        return! checkResultForFile.GetCheckResults.GetSymbolUseAtLocation(pos.Line, pos.Column, lineStr, [])
-      }
+    let getMemberByLocation (name: string, range: Range, keywordRange: Range) =
+      match doc.GetLine range.Start with
+      | Some lineText ->
+        match Lexer.getSymbol range.Start.Line range.Start.Column lineText SymbolLookupKind.ByLongIdent [||] with
+        | Some sym ->
+          checkResultForFile.GetCheckResults.GetSymbolUseAtLocation(
+            range.StartLine,
+            range.EndColumn,
+            lineText,
+            sym.Text.Split('.') |> List.ofArray
```

Comment:
getting the full 'ident islands' here fixed another bug - without this we'd never detect already-implemented members from an abstract class correctly, so our 'partial-fixup' story was broken.